### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by naveen521kk**
## Changelog / Overview

Fix some tests, and cleanup some.

## Motivation
#1556 brought in parallelization of tests which in a way brings new problems with tests. This PR tries to fix those.

## Testing Status
Local pass, windows.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# This file is automatically picked by pytest
```
[the remark this answers is not in the document]
Comment (naveen521kk) [reply conftest.py:1]: yeah, that's the only place pytest looks when invoked for running doctests (I asked on their discord server), or else it should be inside the `manim` module...

[the remark this answers is not in the document]
Comment (behackl) [reply conftest.py:1]: Hm. Okay, thanks for checking back with them. In that case everything here looks good to me!

**Review by behackl — APPROVED** [review]

File: tests/test_plugins/test_plugins.py [hunk 2]
```diff
@@ -64,14 +68,18 @@ def cfg_file_create(cfg_file_contents, path):
...
-def test_plugin_warning(tmp_path, python_version):
-    cfg_file_contents = textwrap.dedent(
-        """
-        [CLI]
-        plugins = DNEPlugin
-    """
+@pytest.fixture
+def random_string():
+    all_letters = string.ascii_lowercase
+    a = random.Random()
```
Comment (huguesdevimeux) [inline tests/test_plugins/test_plugins.py:74]: **Comment** : I could be cool to use a seed in a constant here (that could be by default None) so one can reproduce a test suite.

File: tests/test_plugins/test_plugins.py [hunk 3]
```diff
@@ -64,14 +68,18 @@ def cfg_file_create(cfg_file_contents, path):
...
+    final_letters = [a.choice(all_letters) for _ in range(8)]
+    yield "".join(final_letters)
```
Comment (huguesdevimeux) [inline tests/test_plugins/test_plugins.py:76]: IIRC you can just return it

**Review by huguesdevimeux — APPROVED** [review]
Please check the comments I put, but LGTM in geenral

File: tests/test_plugins/test_plugins.py (hunk 3 above)
Comment (naveen521kk) [reply tests/test_plugins/test_plugins.py:76]: hum, yielding it is a problem?

File: tests/test_plugins/test_plugins.py (hunk 2 above)
Comment (naveen521kk) [reply tests/test_plugins/test_plugins.py:74]: having it as a parameter may be difficult with how fixtures work

**Comment by behackl** [comment]
I will merge this now so that the tests are fixed; further polishing can take place in a follow-up PR. (Sorry if this feels rushed, @huguesdevimeux )

**Pull request merged**